djstripe/models/__init__.py

```diff
@@ -4,7 +4,6 @@
 	Invoice,
 	InvoiceItem,
 	Plan,
-	Product,
```

Comment:
I'm confused why this moved

Reply:
It's a bug fix - the class is in core, the was previously working because it was importing the import.

My changes removed the need for Product in billing so it was failing flake8 due to an unused import without this change.

Reply:
Product originates from core resources, I see the logic here. Product first becomes either good or service, and is interpreted as a plan that, with price and when purchased, comprises a subscription. 

I'm in agreement with the underlying logic - a product is a dumb object first, well before it becomes a billing instrument.

Reply:
Ah, comments in core.py state the case succinctly. core.py:1278-1279 - there's a clear distinction between products and service_products.